lms/envs/common.py

```diff
@@ -2174,6 +2174,13 @@
     'DEFAULT_PAGINATION_CLASS': 'openedx.core.lib.api.paginators.DefaultPagination',
     'PAGE_SIZE': 10,
     'URL_FORMAT_OVERRIDE': None,
+    'DEFAULT_THROTTLE_CLASSES': (
```

Comment:
Yes, since I defined `rest_framework.throttling.UserRateThrottle` as the default throttle, if API views do not override `throttle_classes` attribute these throttles rates will be used. 
I will remove that.